classes/Http/Controller/Reviewer/TalksController.php

```diff
@@ -71,8 +72,12 @@ public function viewAction(Request $req)
 
             return $this->app->redirect($this->url('admin_talks'));
         }
+        $data = [
+                'ratingsystem' => $this->service(TalkRatingStrategy::class)->getRatingName(),
+                'talk'         => $handler->getProfile(),
+            ];
 
-        return $this->render('reviewer/talks/view.twig', ['talk' => $handler->getProfile()]);
+        return $this->render('reviewer/talks/view.twig', $data);
```

Comment:
Maybe we can inline `$data` instead?

```php
return $this->render('admin/talks/view.twig', [
    'talk'         => $handler->getProfile(),
    'ratingsystem' => $this->service(TalkRatingStrategy::class)->getRatingName(),
]);
```

And maybe - for consistency - we can keep the same order as elsewhere?